README.md

```diff
@@ -21,7 +21,9 @@ If you have access to other protocol repos, you can get the values from those re
   
```

Comment:
We now have the dummy.env file so we might not need to include example env values in the README